lib/formatters/grep.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview grep-style formatter.
+ * @author Nicholas C. Zakas
+ */
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Helper Functions
+//-----------------------------------------------------------------------------
+
+function getMessageType(message) {
```

Comment:
The function is actually part of lib/formatters/compact.js which you wrote, could you please provide the JSDoc for it and/or should I write it?